arm-cognitiveservices/2016-02-01-preview/swagger/cognitiveservices.json

```diff
@@ -238,7 +254,7 @@
           "CognitiveServicesAccounts"
         ],
         "description": "Returns all the resources of a particular type belonging to a resource group",
-        "operationId": "CognitiveServicesAccounts_ListByResourceGroup",
+        "operationId": "Accounts_ListByResourceGroup",
```

Comment:
{
    "type": "Error",
    "code": "ListByOperationsValidation",
    "message": "Operation CognitiveServicesAccounts_ListByResourceGroup must be one of: accounts_listbysubscriptionid, accounts_list, accounts_listbyresourcegroup, accounts_listbymicrosoftcognitiveservices",
    "jsonref": "https://raw.githubusercontent.com/Azure/azure-rest-api-specs/master/arm-cognitiveservices/2016-02-01-preview/swagger/cognitiveservices.json#/paths",
    "json-path": "https://raw.githubusercontent.com/Azure/azure-rest-api-specs/master/arm-cognitiveservices/2016-02-01-preview/swagger/cognitiveservices.json#/paths",
    "id": "M1003",
    "validationCategory": "SDKViolation",
    "providerNamespace": null,
    "resourceType": null
  },
  {
    "type": "Error",
    "code": "ListByOperationsValidation",
    "message": "Operation CognitiveServicesAccounts_List must be one of: accounts_listbysubscriptionid, accounts_list",
    "jsonref": "https://raw.githubusercontent.com/Azure/azure-rest-api-specs/master/arm-cognitiveservices/2016-02-01-preview/swagger/cognitiveservices.json#/paths",
    "json-path": "https://raw.githubusercontent.com/Azure/azure-rest-api-specs/master/arm-cognitiveservices/2016-02-01-preview/swagger/cognitiveservices.json#/paths",
    "id": "M1003",
    "validationCategory": "SDKViolation",
    "providerNamespace": null,
    "resourceType": null
  }